ggml/src/ggml-rpc/ggml-rpc.cpp

```diff
@@ -106,13 +106,16 @@ enum rpc_cmd {
     RPC_CMD_GET_ALLOC_SIZE,
     RPC_CMD_HELLO,
     RPC_CMD_DEVICE_COUNT,
+    RPC_CMD_GRAPH_COMPUTE_AND_STORE,
+    RPC_CMD_GRAPH_RECOMPUTE,
     RPC_CMD_COUNT,
 };
 
 static_assert(RPC_CMD_HELLO == 14, "RPC_CMD_HELLO must be always 14");
 
 // Try RPC_CMD_SET_TENSOR_HASH first when data size is larger than this threshold
-const size_t HASH_THRESHOLD = 10 * 1024 * 1024;
+const size_t   HASH_THRESHOLD    = 10 * 1024 * 1024;
+const int      MAX_STORED_GRAPHS = 64;
```

Comment:
Not sure if storing so many graphs is not an overkill. I think just keeping the last graph should be enough.